orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/model/JenkinsTrigger.kt

```diff
@@ -57,7 +57,8 @@ constructor(@param:JsonProperty("name") override val name: String?,
             @param:JsonProperty("result") override val result: String?,
             @param:JsonProperty("artifacts") override val artifacts: List<JenkinsArtifact>?,
             @param:JsonProperty("scm") override val scm: List<SourceControl>?,
-            @param:JsonProperty("building") override var building: Boolean = false)
+            @param:JsonProperty("building") override var building: Boolean = false,
+            @param:JsonProperty("timestamp") val timestamp: Long?)
```

Comment:
Is it really nullable or will Jenkins always supply one?

Reply:
I don’t really know if Jenkins supplies this field at all... Travis always supplies it.